ansible/roles/haproxy/tasks/config.yml

```diff
@@ -125,6 +125,29 @@
   notify:
     - Restart haproxy container
 
+- name: Checking if haproxy-extra certificates dir exists
+  run_once: true
+  local_action: stat path={{ haproxy_extra_cert_directory }}
```

Comment:
I seem to recall `delegate_to: localhost` being preferred over local action in kolla-ansible: https://github.com/openstack/kolla-ansible/commit/25e98f9047165908cbfb6b12c993de179f04b56b